contents/docs/data-warehouse/sql/variables.mdx

```diff
@@ -8,44 +8,63 @@ availability:
   enterprise: full
 ---
 
-export const WarehouseLight = "https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_02_24_at_14_05_39_2x_5a280bacfd.png"
-export const WarehouseDark = "https://res.cloudinary.com/dmukukwp6/image/upload/Clean_Shot_2025_02_24_at_14_05_53_2x_32a95cbe79.png"
+SQL variables enable you to dynamically set values in your queries. 
 
-SQL variables enable you to create configurable, reusable queries by defining placeholders that can be dynamically replaced with different values.
+## Creating SQL variables
 
-## Using variables in SQL queries
+To create a variable, go to the [SQL editor](https://app.posthog.com/sql), click the **Variables** tab, click **Add variable**, and choose your variable type (string, number, date, list, or boolean). Enter the variable name and value(s) and click **Save** and you'll be able to use it in your queries.
 
-You must first create a variable in the variable sidebar panel. Once created, variables in PostHog SQL are defined using the `{variables.<variable-name>}` syntax. When you write a query with variables, you'll be prompted to provide values for each variable.
+<ProductScreenshot
+  imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/Clean_Shot_2025_10_02_at_17_04_16_2x_aac766c379.png"
```

Comment:
Wait how do you do variables with events? This screen shot looks like variables with events

Reply:
OH that's just the variable name?

Reply:
added, yes